src/app/plugins/checkclasses.plugin.ts

```diff
@@ -15,6 +15,10 @@ export default class CheckClassesPlugin extends Plugin {
 
   private _MAX_CHANS_SHOWN: number = 10;
 
+  public override validate(message: IMessage, args: string[]) {
+    return args.length != 0 && !!Moderation.Helpers.validateUser(args[0]);
```

Comment:
why are you !! On a function call that returns Boolean

Reply:
lol you did tell them to do that, but yeah it should be removed, good catch kek